rpcs3/Emu/Cell/lv2/sys_usbd.cpp

```diff
@@ -525,24 +580,40 @@ usb_handler_thread::~usb_handler_thread()
 	handled_devices.clear();
 	open_pipes.clear();
 	usb_devices.clear();
+	usb_passthrough_devices.clear();
 
 	for (u32 index = 0; index < MAX_SYS_USBD_TRANSFERS; index++)
 	{
 		if (transfers[index].transfer)
 			libusb_free_transfer(transfers[index].transfer);
 	}
 
+#if LIBUSB_API_VERSION >= 0x01000102
+	libusb_hotplug_deregister_callback(ctx, callback_handle);
+#endif
+
 	if (ctx)
 		libusb_exit(ctx);
 }
 
 void usb_handler_thread::operator()()
 {
 	timeval lusb_tv{0, 0};
-
+	if (!hotplug_supported)
+	{
+		usb_hotplug_timeout = get_system_time() + 4'000'000ull;
+	}
 	while (ctx && thread_ctrl::state() != thread_state::aborting)
 	{
-		// Todo: Hotplug here?
+		const u64 now = get_system_time();
+		if (usb_hotplug_timeout && now > usb_hotplug_timeout)
```

Comment:
I think this should always be umax if disabled and then:
```suggestion
		if (now > usb_hotplug_timeout)
```
Kinda makes it easier to understand for me